### PR TITLE
Updates for CD pipeline

### PR DESCRIPTION
**Pull request by VladimirWork**

### CODE REVIEW

File: system/utils.py [hunk 1]
```diff
@@ -136,6 +136,9 @@ def random_seed_and_json():
                                  "remaining..., will sleep for {}".format(fname, remain, sleep_dur))
                 await asyncio.sleep(sleep_dur)
             else:
+                print("\n{} failed; not trying any more because {} "
+                      "seconds have passed; args were {}".
+                      format(fname, timeout, args))
```
Comment (andkononykhin) [inline system/utils.py]: why do you think you need this? `pytest` provides ability to pass logs to stdout using `--log-cli-level` option

File: system/indy-node-tests/test_vc.py [hunk 2]
```diff
@@ -12,45 +12,42 @@
...
+    await ensure_pool_is_functional(pool_handler, wallet_handler, trustee_did)
     p1.start_service()
-    await eventually_positive(check_pool_is_in_sync)
+    await ensure_pool_is_in_sync(nodes_num=nodes_num)
+    await ensure_pool_is_functional(pool_handler, wallet_handler, trustee_did)
 
 
 @pytest.mark.asyncio
-async def test_vc_by_demotion(pool_handler, wallet_handler, get_default_trustee):
+async def test_vc_by_demotion(
```
Comment (andkononykhin) [inline system/indy-node-tests/test_vc.py]: You might consider to use `check_no_failures_fixture` here as well.

File: system/indy-node-tests/test_vc.py [hunk 3]
```diff
@@ -12,45 +12,42 @@
...
 @pytest.fixture(scope='function', autouse=True)
 @async_generator
 async def docker_setup_and_teardown(nodes_num):
     await setup_and_teardown(nodes_num)
 
 
 @pytest.mark.asyncio
-async def test_vc_by_restart(pool_handler, wallet_handler, get_default_trustee):
+async def test_vc_by_restart(
+        pool_handler, wallet_handler, get_default_trustee, nodes_num
```
Comment (andkononykhin) [inline system/indy-node-tests/test_vc.py]: You might consider to use `check_no_failures_fixture` to monitor node services failures to interrupt test in earlier phase if they happen.

File: system/indy-node-tests/TestMultiSigSuite.py [hunk 4]
```diff
@@ -43,8 +43,9 @@ class TestMultiSigSuite:
         trustee_did, _ = get_default_trustee
         new_did, new_vk = await did.create_and_store_my_did(wallet_handler, '{}')
         req = await ledger.build_nym_request(trustee_did, new_did, new_vk, random_string(5), None)
-        req = await ledger.sign_request(wallet_handler, trustee_did, req)
+        # if order of sign and multisign will be changed - test fails
```
Comment (andkononykhin) [inline system/indy-node-tests/TestMultiSigSuite.py]: Don't you think that the test is ready for activation (I mean removing `skip` mark)?

File: system/draft/test_misc.py [hunk 5]
```diff
@@ -802,7 +802,7 @@ def test_misc_hypothesis_sync(docker_setup_and_teardown, pool_handler, wallet_ha
 async def test_misc_mint_to_aws(payment_init):
     await pool.set_protocol_version(2)
     libsovtoken_payment_method = 'sov'
-    pool_handle, _ = await pool_helper(path_to_genesis='../aws_genesis_test')
+    pool_handle, _ = await pool_helper(path_to_genesis='../aws_genesis')
```
Comment (andkononykhin) [inline system/draft/test_misc.py]: `../aws_genesis`? Is it something that should be committed to the repo?

File: system/draft/TestAuditSuite.py [hunk 6]
```diff
@@ -5,55 +5,64 @@
...
 class TestAuditSuite:
 
     @pytest.mark.asyncio
-    async def test_case_restart_one_node(self, pool_handler, wallet_handler, get_default_trustee):
+    async def test_case_restart_one_node(
+            self, pool_handler, wallet_handler, get_default_trustee, nodes_num
+    ):
         trustee_did, _ = get_default_trustee
         test_nodes = [NodeHost(i) for i in range(1, 8)]
+        # ensure?
```
Comment (andkononykhin) [inline system/draft/TestAuditSuite.py]: Why this question is here (and similar questions lower)? What possible improvement is implied?

File: system/utils.py [hunk 7]
```diff
@@ -806,6 +808,82 @@ def get_node_did(node_alias, pool_info=None):
     return is_exception_raised
 
 
+# TODO remove this eventually and all custom eventuallies above later
```
Comment (andkononykhin) [inline system/utils.py]: I don't see any usage of that helper? Do you think it is necessary in scope of the current PR?

File: system/utils.py (hunk 1 above)
Comment (VladimirWork) [reply system/utils.py]: Removed.

File: system/indy-node-tests/test_vc.py (hunk 2 above)
Comment (VladimirWork) [reply system/indy-node-tests/test_vc.py]: Done.

File: system/indy-node-tests/test_vc.py (hunk 3 above)
Comment (VladimirWork) [reply system/indy-node-tests/test_vc.py]: Done.

File: system/indy-node-tests/TestMultiSigSuite.py (hunk 4 above)
Comment (VladimirWork) [reply system/indy-node-tests/TestMultiSigSuite.py]: Removed.

File: system/utils.py (hunk 7 above)
Comment (VladimirWork) [reply system/utils.py]: Removed.

File: system/draft/test_misc.py (hunk 5 above)
Comment (VladimirWork) [reply system/draft/test_misc.py]: I believe that we don't need to run any system tests in CD against AWS pool so no.

File: system/draft/TestAuditSuite.py (hunk 6 above)
Comment (VladimirWork) [reply system/draft/TestAuditSuite.py]: I've replaced common nyms sending with new ensure helpers. Test run time remains pretty the same.

**Pull request closed without merging**